user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -107,29 +111,25 @@ and change `/dev/sdY` to the correct target device e.g., `/dev/sdc`). Make sure
 to write to the entire device (e.g., `/dev/sdc`) rather than just a single
 partition (e.g., `/dev/sdc1`).
 
-On Windows, you can use the [Rufus](https://rufus.akeo.ie/) tool to write the
-ISO to a USB key. MediaTest is not recommended. Be sure to select "DD image"
-mode (*after* selecting the Qubes ISO):
+#### Windows ISO to USB
 
-<div class="alert alert-warning" role="alert">
-  <i class="fa fa-exclamation-circle"></i>
-  <b>Note:</b>  If you do this on Windows 10, you can only install Qubes
-  without MediaTest, which is not recommended.
-</div>
+On Windows, you can use the [Rufus](https://rufus.akeo.ie/) tool to write the
+ISO to a USB key. Be sure to select "Write in DD Image mode" *after* selecting
+the Qubes ISO:
 
```

Comment:
I dont understand.
You've removed the reference to testing the media altogether. Is it no longer true that even in dd mode, Windows will alter the media - QubesOS/qubes-issues#2051 ?
If users who have used Windows cant use "Test this media" during installation, you should explicitly say so.

Also, although the instructions refer to a variety of installation media, they dont explain what to do  for a DVD or disc. Can you include a reference? Like "Adapt these instructions if you are writing to a DVD"

Reply:
> If users who have used Windows cant use "Test this media" during installation, you should explicitly say so.

Thanks, @unman!  This sentence solved my "MediaTest" confusion that was the reason for my initial forum post.

Reply:
@unman, I believe this PR now addresses the changes you requested. Let me know if you want further change.